lumps/fdm_deh.lmp

```diff
@@ -75,31 +75,31 @@ par 32 0
 STARTUP5============================================================================\nThis is FreeDM, the free content deathmatch FPS.\n\nFreeDM is freely redistributable under the terms of the modified BSD\nlicense. Check out the Freedoom website for more information:\n\n    https://freedoom.github.io/\n============================================================================
 GOTARMOR = picked up a light armor vest.
 GOTMEGA = picked up a heavy armor vest.
-GOTHTHBONUS = got a 1% health bonus.
-GOTARMBONUS = got a 1% armor bonus.
+GOTHTHBONUS = got a vitality boost.
```

Comment:
I'm going to have to push back on this. Is there a reason to use "vitality" instead of "health" or "shield" instead of "armor"? This is terminology common across almost all FPS games (and non-FPS games too). At the very least we should be consistent - why should it be called "health" on the status bar but "vitality" in the messages? It's needlessly confusing and I don't understand why you want to change it.